feconf.py

```diff
@@ -403,6 +403,13 @@ def get_empty_ratings():
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
 ENABLE_STRING_CLASSIFIER = False
+SHOW_COLLECTION_NAVIGATION_TAB_MAIN = False
+SHOW_COLLECTION_NAVIGATION_TAB_PREVIEW = False
+SHOW_COLLECTION_NAVIGATION_TAB_SETTINGS = False
+SHOW_COLLECTION_NAVIGATION_TAB_HISTORY = False
+SHOW_COLLECTION_NAVIGATION_TAB_FEEDBACK = False
+SHOW_COLLECTION_NAVIGATION_TAB_STATS = False
+SHOW_COLLECTION_NAVIGATION_TAB_HELP = False
```

Comment:
This probably isn't needed, since we aren't planning on implementing a help tab currently.